src/Data/RFC5322.hs

```diff
@@ -179,6 +186,20 @@ data Mailbox =
              AddrSpec
     deriving (Show,Eq)
 
+renderMailboxes :: [Mailbox] -> B.ByteString
+renderMailboxes xs = B.intercalate ", " $ renderMailbox <$> xs
+
+renderMailbox :: Mailbox -> B.ByteString
+renderMailbox (Mailbox n a) =
+    let name =
+            maybe
+                mempty
+                (\x ->
+                      mconcat ["\"", Builder.byteString $ T.encodeUtf8 x, "\" "])
+                n
+        builder = mconcat [name, "<", renderAddressSpec a, ">"]
```

Comment:
It is valid to use angle brackets when `name` is empty, but I'd probably prefer to avoid it, e.g.:
```haskell
renderMailbox (Mailbox n a) =
  maybe a' (\n' -> renderDisplayName n' <> " <" <> a' <> ">") n
  where
    a' = renderAddressSpec a
```